### PR TITLE
Removed unused/superfluous/dead code

### PR DESCRIPTION
**Pull request by carloscaldas**
- Removed unused variables
- Transformed fields in local variables
- Removed superflous override method
- Removed dead code

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/716)

### CODE REVIEW

File: src/main/java/beam/analysis/plots/RideHailWaitingStats.java [hunk 1]
```diff
@@ -127,14 +127,13 @@ private Double getCategory(double time, List<Double> categories) {
     private int rideHailCount = 0;   //later used to calculate average wait time experienced by customers
     private final IStatComputation<Tuple<List<Double>, Map<Integer, List<Double>>>, Tuple<Map<Integer, Map<Double, Integer>>, double[][]>> statComputation;
 
-    private int timeBinSize = 3600;
```
Comment (JustinPihony) [inline src/main/java/beam/analysis/plots/RideHailWaitingStats.java]: This is already in #675

File: src/main/java/beam/analysis/via/EventWriterXML_viaCompatible.java [hunk 2]
```diff
@@ -26,10 +26,8 @@
     private boolean eventsForFullVersionOfVia;
     HashMap<String, HashSet<String>> filterPeopleForViaDemo = new HashMap<>();
     HashMap<String, Integer> maxPeopleForViaDemo = new HashMap<>();
-    private String outFileName;
```
Comment (JustinPihony) [inline src/main/java/beam/analysis/via/EventWriterXML_viaCompatible.java]: Same - already in #675

File: src/main/java/beam/playground/beamSimAkkaProtoType/beamPersonAgent/BeamPersonAgent.java [hunk 3]
```diff
@@ -61,10 +60,6 @@ public void onReceive(TriggerMessage message) {
             // new PersonArrivalAtVehicleMessage(getSelf(), arrivalTime, 0));
 
         } else if (message instanceof PersonArrivalAtVehicleMessage) {
-            if (vehicleIsPluggedIn) {
```
Comment (JustinPihony) [inline src/main/java/beam/playground/beamSimAkkaProtoType/beamPersonAgent/BeamPersonAgent.java]: @colinsheppard - this seems a bit EV-ish, so you may want to verify whether this should be removed, or fixed?

File: src/main/java/beam/router/r5/OsmToMATSim.java [hunk 4]
```diff
@@ -51,7 +50,6 @@
     public OsmToMATSim(final Network mNetwork, final CoordinateTransformation transformation,
```
Comment (JustinPihony) [inline src/main/java/beam/router/r5/OsmToMATSim.java]: Can we just remove this from the constructor?

File: src/main/java/beam/utils/BeamCalcLinkStats.java [hunk 5]
```diff
@@ -48,15 +48,13 @@
...
     private int count = 0;
-    private double minCarSpeedInMetersPerSecond;
 
     @Inject
     public BeamCalcLinkStats(final Network network, double minCarSpeedInMetersPerSecond) {
         this.network = network;
         this.linkData = new ConcurrentHashMap<>();
         this.nofHours = 24;
         reset();
-        this.minCarSpeedInMetersPerSecond = minCarSpeedInMetersPerSecond;
```
Comment (JustinPihony) [inline src/main/java/beam/utils/BeamCalcLinkStats.java]: Can we just remove the parameter from the constructor?

File: src/main/java/beam/router/r5/OsmToMATSim.java (hunk 4 above)
Comment (carloscaldas) [reply src/main/java/beam/router/r5/OsmToMATSim.java]: yes

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

**Pull request closed without merging**